anilist.py

```diff
@@ -155,45 +155,45 @@ def _make_request(query: str, variables: dict):
         print(f'429: {response}. Waiting {timeout_seconds} seconds...', file=sys.stderr)
         time.sleep(timeout_seconds)
 
+    time.sleep(0.7) #This should _theoretically_ mean we never hit the 429 again.
     return response.json()
 
 def get_media_users_are_ineligible_for(user_ids: list[int], media_ids: set[int]) -> defaultdict[int, list[int]]:
     # Sorting for caching
-    return _get_media_users_are_ineligible_for(sorted(user_ids), sorted(media_ids))
+    data = _get_media_users_are_ineligible_for(sorted(user_ids), sorted(media_ids))
+
+    # Default dicts are great.
+    user_dict = defaultdict(list)
+    for list_item in data:
+        if list_item["status"] == 'PLANNING': continue
```

Comment:
The problem child that was assigning users things they'd seen was this == used to be a != and was skipping at the wrong time. (see old file line 196). I also lifted this part of the function to this level  to make the "calling" function just call the API and not be doing two things.